Libs/DICOM/Widgets/ctkDICOMTableView.cpp

```diff
@@ -146,7 +155,22 @@ void ctkDICOMTableViewPrivate::hideUIDColumns()
 //----------------------------------------------------------------------------
 QString ctkDICOMTableViewPrivate::queryTableName() const
 {
-  return this->lblTableName->text();
+  //return this->lblTableName->text();
```

Comment:
STYLE: remove dead code
STYLE: change to `labelTableName` (I see this was already there and you were being consistent but it would be good clean this up while we are working on this code)

Reply:
OK